charts/rollups-validator-node/values.yaml

```diff
@@ -587,32 +663,39 @@ endpoints:
       ##   cpu: 100m
       ##   memory: 128Mi
 
-    ## endpoints.graphqlServer.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the endpoints.graphqlServer container(s)
-    ##
+    # -- endpoints.graphqlServer.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the endpoints.graphqlServer container(s)
     extraVolumeMounts: []
 
 ## redis configuration
```

Comment:
Could we add a link to the original charts documentation?

We're only exposing the values we use here, but if the user needs to change any other parameter, they could see the available options.

Reply:
I mean, for redis and postgresql.

Reply:
The links are added in the requirement section